tests/slr_integration/Admin/__snapshots__/Maps_Layout_Homepage_Test__test_layouts_tab_card_listing__0.snapshot.html

```diff
@@ -39,14 +39,14 @@
 	</div>
 	<div class="tec-tickets__seating-tab__card-content">
 		<div class="tec-tickets__seating-tab__card-title">Layout 1</div>
-		<div class="tec-tickets__seating-tab__card-capacity">
-			100 seats		</div>
+		<div class="tec-tickets__seating-tab__card-info">
+			No associated events		</div>
```

Comment:
Here and elsewhere in the tests, the tets case should be setting things up to have Events associated with the map, but the map is showing as not associated to anything.

Reply:
The Maps card doesn't have any associated events [showing](https://www.figma.com/design/5TiznOUTaiwQQ5FZAgwRDm/Seat-Charts?node-id=2503-180691).

We only display associated events for [Layout Cards.](https://www.figma.com/design/5TiznOUTaiwQQ5FZAgwRDm/Seat-Charts?node-id=4243-299350)

The highlighted text is from the layouts tab -> Layout card. The maps view is not changed with this PR.